src/cli/commands/server.rs

```diff
@@ -0,0 +1,38 @@
+//! The Daemon command line handler that prints the info about IPC Agent.
+
+use async_trait::async_trait;
+use clap::Args;
+use std::fmt::Debug;
+
+use crate::cli::CommandLineHandler;
+use crate::config::Config;
+use crate::server::jsonrpc::JsonRPCServer;
+
+/// The command to start the ipc agent json rpc server in the foreground.
+pub(crate) struct LaunchJsonRPC;
+
+#[async_trait]
+impl CommandLineHandler for LaunchJsonRPC {
```

Comment:
Let's call this `LaunchDaemon` or something more generic, in the future we may launch with this CLI more processes, not only the jsonrpc server
```suggestion
impl CommandLineHandler for LaunchDaemon {
```

Reply:
updated, resolving this